### PR TITLE
Rebellion Theme Release V1.0

### PR DESCRIPTION
**Pull request by marjohn56**
Updated for new jquery.tokenize

### CODE REVIEW

File: misc/theme-rebellion/src/opnsense/www/themes/rebellion/build/css/jquery.tokenize [hunk 1]
```diff
@@ -0,0 +1,171 @@
+div.TokenizeMeasure,
```
Comment (fichtner) [inline misc/theme-rebellion/src/opnsense/www/themes/rebellion/build/css/jquery.tokenize]: mhhh, new file, not moved over .css :)

**Comment by marjohn56** [comment]
Indeed... another one to be moved. :)

**Comment by fichtner** [comment]
Did it manually in 5a72ca0 :)

Thanks!

**Comment by marjohn56** [comment]
Confused ,,, as always, but as long as it works. :)

**Pull request closed without merging**